README.md

```diff
@@ -1,26 +1,65 @@
+
 <p align="center"> 
-  <img src="https://i.imgur.com/UK3PWJW.png" width="175px" height="175px">
+  <img src="https://i.imgur.com/AohWq5l.png">
 </p>
 
-# Quaver [![Build Status](https://travis-ci.com/Quaver/Quaver.svg?branch=develop)](https://travis-ci.com/Quaver/Quaver) [![CodeFactor](https://www.codefactor.io/repository/github/swan/quaver/badge)](https://www.codefactor.io/repository/github/swan/quaver) [![Blog](https://img.shields.io/badge/Blog-Read-blue.svg)](https://blog.quavergame.com) [![Discord](https://discordapp.com/api/guilds/354206121386573824/widget.png?style=shield)](https://discord.gg/nJa8VFr)
+| [![steam](https://i.imgur.com/rR4p9mW.png)](https://store.steampowered.com/app/980610/Quaver/) | [![twitter](https://i.imgur.com/Kd0SXlh.png)](https://twitter.com/QuaverGame) | [![discord](https://i.imgur.com/YrBmbEf.png)](https://discord.gg/nJa8VFr) | [![blog](https://i.imgur.com/o0fq6MA.png)](https://blog.quavergame.com) | [![website](https://i.imgur.com/svZCnuI.png)](https://quavergame.com) | 
+| --- | --- | --- | --- | --- |
+| **Steam Store Page** | **Follow Us On Twitter** | **Join The Discord** | **Read The Developer Blog** | **Visit The Website** |
+
+## About 
+
+**Quaver** is a community-driven and open-source competitve vertical scrolling rhythm game with two game modes and online leaderboards. It also features an in-game editor and a flexible multiplayer mode.
+
+It is also being officially released on [Steam](https://store.steampowered.com/app/980610/Quaver/) for Windows, Mac, and Linux - making it one of the most accessible community-driven rhythm games to date.
+
+## Status
+
+This project is currently under heavy development and is in an alpha state, however it is extremely playable as a standalone game. Quaver still has a ton of development ahead of it and will continue to get better as time goes on with new features and bug fixes being added in each day.
+
+Occasionally we open registrations where you can join the Steam alpha and get exclusive access to all the features Quaver has to offer including online rankings, multiplayer, in-game chat, map downloading/uploading, and so much more. Registrations are **currently closed,** however you can still enjoy a limited version of Quaver by compiling the `develop` branch or by downloading one of the [GitHub Releases](https://github.com/Quaver/Quaver/releases).
+
+## Features
+
+Although there are many games like this, **Quaver** offers many unique features and expands on the ideas of its predecessors that make it much different. It is also heavily inspired by a variety of our favorite rhythm games.
+
+* **Two Game Modes** - Play the game with 4 keys or challenge yourself with 7. Each game mode has separate global and country leaderboards to compete on.
+
+* **Global & Country Leaderboards** - Compete with your country and the world for the highest scores and aim to be #1.
```

Comment:
I see what you mean by the word "scores" but it sounds funny considering that score is irrelevant for this